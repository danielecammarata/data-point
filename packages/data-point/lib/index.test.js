/* eslint-env jest */

const lib = require("./index");

describe("lib", () => {
  // this test here is mainly to be aware of any new method that gets added
  // to the main API
  test("It should explicity expose set of methods", () => {
    expect(lib).toMatchSnapshot();
  });
});
