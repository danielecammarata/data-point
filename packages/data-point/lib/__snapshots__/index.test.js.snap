// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lib It should explicity expose set of methods 1`] = `
Object {
  "Collection": [Function],
  "Control": [Function],
  "Entry": [Function],
  "Hash": [Function],
  "Model": [Function],
  "Reducer": [Function],
  "Request": [Function],
  "Schema": [Function],
  "assign": [Function],
  "constant": [Function],
  "create": [Function],
  "createEntity": [Function],
  "createReducer": [Function],
  "createReducerResolver": [Function],
  "createTypeCheckReducer": [Function],
  "entities": Object {
    "Collection": [Function],
    "Control": [Function],
    "Entry": [Function],
    "Hash": [Function],
    "Model": [Function],
    "Reducer": [Function],
    "Request": [Function],
    "Schema": [Function],
  },
  "entityFactories": Object {
    "Collection": [Function],
    "Control": [Function],
    "Entry": [Function],
    "Hash": [Function],
    "Model": [Function],
    "Reducer": [Function],
    "Request": [Function],
    "Schema": [Function],
  },
  "filter": [Function],
  "find": [Function],
  "helpers": Object {
    "assign": [Function],
    "constant": [Function],
    "filter": [Function],
    "find": [Function],
    "map": [Function],
    "omit": [Function],
    "parallel": [Function],
    "pick": [Function],
    "withDefault": [Function],
  },
  "map": [Function],
  "omit": [Function],
  "parallel": [Function],
  "pick": [Function],
  "reducify": [Function],
  "reducifyAll": [Function],
  "resolveEntity": [Function],
  "withDefault": [Function],
}
`;
